### ISSUE TITLE
[BUG] Lightning Launcher doesn't remember grouping.

### ISSUE THREAD

**Issue by gaveitatry2** [threethan/LightningLauncher#65]
I've noticed this for around a month or however long it's been since I first started using Lightning Launcher. I'd move a few games to a certain group. And then at a later date, I'd notice that those games aren't in that group I wanted but are back in the original group, even though there has been no updates to the games themselves. Recently, in the last week, I moved two Dance games and a Kung Fu game from group 003 to group 002. I noticed at a later date that they were back in group 003 again. So I moved them again. Then later once again they were back in 003. So I just moved them a third time. I bet that eventually the Lightning Launcher app will remember and keep it in the right group (because when this first happened to me with different games, it eventually remembered the right group), but it's still annoying that it does this. I have only installed Lightning Launcher once. I don't know if it auto updates, but if it does then I should be on the newest version. If it doesn't auto update, then it's probably a one month old release.

**Comment by gaveitatry2** [comment]
It happened another two times since yesterday, and I barely even touched my Quest 3 at all. So that makes five times so far that the same three apps won't remember what group I want them in.

**Comment by cjk87927** [comment]
How to move an application to a custom group?

**Comment by gaveitatry2** [comment]
> How to move an application to a custom group?

You just long press on any app. This is a quick way to get into Edit mode. Then you click which apps you want to select for moving. You can select more than one app at a time. When they are selected, they will be grayed out which makes it easier to tell from unselected apps. Then click the group tab that you want to move them to and they will immediately move to that group. Then you click the Stop Editing button on the bottom right.

**Comment by gaveitatry2** [comment]
@threethan - Have you seen this issue yet? The problem still exists. I keep moving the same two Dance apps and Kung Fu game from my 003 group to my 002 group. I save my changed. And then anytime I open Lighting Launcher in the future, like a day later, they are back in the 003 group again. I've had to move them 20 times already. Both 003 and 002 groups do not have any "default apps" checked. I just use them to drag certain types of games and apps in as a separate category.

**Comment by threethan** [comment]
That's odd. Did you manually name your groups like that (002, 003)? In all versions of LL, new groups should be named New, New 2, New 3, etc...

Where did you initially download the launcher from? There was some hacked together half-russian version of LL floating around a while back.

It seems like there may be some sort of corruption to the shared storage of LL. Some older versions had occasional issues with this. I would try uninstalling and reinstalling the launcher to see if this fixes it.

**Comment by gaveitatry2** [comment]
> That's odd. Did you manually name your groups like that (002, 003)? In all versions of LL, new groups should be named New, New 2, New 3, etc...
> 
> Where did you initially download the launcher from? There was some hacked together half-russian version of LL floating around a while back.
> 
> It seems like there may be some sort of corruption to the shared storage of LL. Some older versions had occasional issues with this. I would try uninstalling and reinstalling the launcher to see if this fixes it.

Yes, of course I renamed the groups and created new ones. Those are features in Lightning Launcher that I used. It's not like I hacked your program. I just didn't like the default names of the groups.

I downloaded the launcher from SideQuest.

**Comment by gaveitatry2** [comment]
I will come back to you on this. I don't think you will be satisfied until you have given me the biggest headache possible, wasted as much of my time as possible, and blow as much smoke out your behind as possible.

I uninstalled the app for now. I took some screenshots that I will share with you at a later time. I will eventually reinstall the app from Sidequest again, and give it another week or so to test it out to see if the grouping is still broken.

I'm not in a hurry to do it because I haven't felt like using VR this week and also just now I restarted my headset (even though I've restarted it yesterday and the day before AFTER the 60.0 update), and suddenly my room boundary is messed up. And I don't feel like redrawing it again because it covers multiple rooms and it takes time to draw.

Anyway, thanks for nothing. Will message you about this again in a couple of weeks.
